web_src/js/components/RepoActionView.vue

```diff
@@ -802,10 +823,15 @@ export function initRepositoryActionView() {
   display: flex;
 }
 
-.job-step-section .job-step-logs .job-log-line:hover {
+.job-step-section .job-step-logs .job-log-line:hover,
+.job-log-line:target {
```

Comment:
The styles are scoped, so it could be as simple as:

```suggestion
.job-log-line:hover,
.job-log-line:target {
```